ssn/integrated/informativeExamples/InkBird-IBS-TH2-instance.ttl

```diff
@@ -0,0 +1,224 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix ssn: <http://www.w3.org/ns/ssn/> .
+@prefix ssn-system: <http://www.w3.org/ns/ssn/systems/> .
+@prefix ex: <http://example.org/> .
+
+@base <http://example.org/data/> .
+
+# This example represents a specific InkBird IBS-TH2 sensor that is performing
+# measurements in multiple locations where the results are owned by different
+# entities.
+
+<acmeBreweryCo> a org:Organization, beer:Brewery ;
+	foaf:name "Acme Brewery Co." .
+
+<acmeBrewerCooler> a equipment:Cooler .
+
+<alice> a foaf:Person, prov:Agent ;
+	foaf:name "Alice" .
+
+<acmeDelivery> a org:Organization ;
+	foaf:name "Acme Delivery Co." .
+
+<acmeTruck> a ex:RefrigeratedTruck .
+
+<acmeSupermarket> a <org:Organization> ;
+	foaf:name "Acme Supermarket Co." .
+
+<acmeSupermarketCooler> a <equipment:Cooler> .
+
+# An instance of a six pack of Porter Beers 
+<00001/acmePorterSixPack> a beer:Porter, gs1:Product ;
+    gs1:lot "00001" .
+
+# Alice packages the Beer.
+
+<12345/someTH2> a equipment:InkBird-IBS-TH2 ;
+    rel:own <acmeBreweryCo> ; # Sensor may be returned.
+    rdfs:label "InkBird Sensor that Alice bought to track beer storage."@en ;
+    sosa:hosts <12345/BatterySensor>, <thisHumiditySensor>, <thisTemperatureSensor> ;
+    gs1:hasSerialNumber "12345" ;
+    equipment:deviceAddress "00:00:00:00:00:00" .
+
+
+<00001/packingSixPack> a prov:Activity ;
+    rdfs:comment "When Alice packaged Porter bottles into the box, she added a InkBird logger to check that the beer wasn't getting too warm in transit and storage." ;    
+    prov:wasAssociatedWith <alice> ;
+    prov:used <someTH2> ;
+    prov:startedAtTime "2024-02-20T01:35:00Z"^^xsd:dateTime;
+    prov:endedAtTime   "2024-02-20T01:40:00Z"^^xsd:dateTime;   
+    prov:atLocation <acmeBreweryCo> ;
+    prov:generated <00001/acmePorterSixPack> .
+
+# These definitions look redundant; but they represent the specific, physical instantiation of the sensors.
+
+
+# Batteries are a consumable component of the platform and must be replaced from time to time. Since the
+# battery health sensor is observing a new battery pack each time it is replaced, we create a new digital
+# twin to be related to the observation. This means that device power consumption and the performance of
+# different battery brands can also be analyzed.
+
+<2024/02/batteryPack> a equipment:IBSTH2Battery;
+    rdfs:label "IBSTH2 Battery. 2 AAA Batteries. 3.0V Nominal."@en ;
+    skos:editorialNote "Alice doesn't quite believe the battery lifetime claims, so she's tracking every set of batteries she uses and a new battery instances is generated each time." ;
+    gs1:brand <AcmeBatteryBrand> ;
+    dcterms:created "2024-02-20T01:35:45Z"^^xsd:dateTime .
+
+<AcmeBatteryBrand> a gs1:Brand ;
+    rdfs:label "Acme Branded Battery"@en ;
+    gs1:brandName "Acme"@en .
+
+<12345/BatterySensor> a equipment:IBSTH2BatteryVoltageSensor;
```

Comment:
The battery pack seems more as a FoI than a Property. The Property could be something like "Charge".

Reply:
I'm using 'health' as the sensor documentation has no information as to what it is actually measuring. Likely, it is a linear regression based on the battery voltage, but this is beyond the scope of the problem. FOI fixed.